rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -216,6 +226,32 @@ def set_enabled(enabled: bool) -> None:
     bindings.set_enabled(enabled)
 
 
+def strict_mode() -> bool:
+    """
+    Strict mode enabled.
+
+    In strict mode, misuses of the Rerun API will result in exception being raised.
+    When stirct mode if oss, such misuses are instead logged as warnings.
```

Comment:
```suggestion
    When strict mode is on, such misuses are instead logged as warnings.
```

this is an intentional "did they actually review this"-trap, right? In so many doc strings you wrote I've never seen you do this 😆

Reply:
strict mode was off on the spell checker today